src/character_controller.cpp

```diff
@@ -45,12 +44,15 @@ void Character_Create(struct entity_s *ent)
         ret->weapon_current_state = 0x00;
         ret->current_weapon = 0;
 
-        ret->resp.vertical_collide = 0x00;
-        ret->resp.horizontal_collide = 0x00;
-        ret->resp.kill = 0x00;
-        ret->resp.burn = 0x00;
-        ret->resp.slide = 0x00;
-        ret->resp.step_z = 0x00;
+        ret->state.vertical_collide = 0x00;
+        ret->state.horizontal_collide = 0x00;
+        ret->state.slide = 0x00;
+        ret->state.step_z = 0x00;
+        ret->state.kill = 0x00;
```

Comment:
Is `kill` a request or a state? If it's a state, I'd rather call it `dying` or `dead`.